### DISCUSSION TITLE
Can I configure renovate to update the indirect dependencies in golang

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [renovatebot/renovate#24032] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Is there a way to configure renovate to check indirect dependencies in go.mod file and generate PR if they have updated?

### What you tried so far.

I read the documentation. Seems like lockFileMaintenance should be doing that, but I do not see it listed.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by lmilbaum** [comment, marked as answer]
Yes. Add the following configuration to your config.js file.
```
  "packageRules": [
    {
      "description": "Enable go indirect dependencies",
      "matchManagers": ["gomod"],
      "matchDepTypes": ["indirect"],
      "enabled": true
    }
  ]
```
